views/pod.html

```diff
@@ -15,7 +15,7 @@
     <dt>Phase</dt>
     <dd>{{resource.status.phase}}</dd>
     <dt>Node</dt>
-    <dd>{{resource.status.host}}</dd>
+    <dd>{{resource.spec.host}}</dd>
```

Comment:
Sorry @liggitt i just read your whole comment.  Forgot this was broken out into the status section in the describer.

It seems weird to say what node it is on in multiple places..  when the host.spec is defined can the node it ends up on ever be different? @smarterclayton